go/client/chat_cli_rendering.go

```diff
@@ -217,10 +217,10 @@ func (v conversationListView) show(g *libkb.GlobalContext, myUsername string, sh
 		// show the last TEXT message
 		var msg *chat1.MessageUnboxed
 		for _, m := range conv.MaxMessages {
-			if conv.ReaderInfo.ReadMsgid < m.GetMessageID() {
-				unread = "*"
```

Comment:
only thing I'm confused about is this change, I might have moved the block by accident but on master the file matches what is here, unsure what is the correct version. Didn't mean to touch this file at all.